.github/workflows/release.yaml

```diff
@@ -73,3 +73,15 @@ jobs:
           nexus_password: ${{ secrets.SONATYPE_PASSWORD }}
           maven_profiles: sign-source-javadoc
           directory: java
+
+  publish-go:
+    name: Create go/* tag
+    needs: create-release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create go/* tag
```

Comment:
Even though it's trivial, I suggest we factor this out into an action too. That way we keep symmetry with the other publish jobs, and we also have a single lever we can move if we need to make cross-project modifications about how Go packages are released (such as publishing binaries etc.)

Reply:
Agree, but I'd prefer to extract it *after* we're sure it works. It's easier to iterate that way.

Reply:
> Agree, but I'd prefer to extract it _after_ we're sure it works. It's easier to iterate that way.

Mmmyeah. Remember how hard these are to test though - really the only way to test it is to run it and make a release. If we have it in a separate action we can unit test that at least, which gives us a bit more confidence when we bolt the whole thing together.

I've run out of time today but I can do this tomorrow.